src/delagent/ui/DeleteMessages.php

```diff
@@ -24,7 +24,7 @@
  */
 abstract class DeleteMessages
 {
-  const NO_PERMISSION = 0; /**< You don't have permissions to delete the upload */
   const SCHEDULING_FAILED = 1; /**< Deletion Scheduling failed */
   const SUCCESS =  2; /**< Deletion added to job queue */
+  const NO_PERMISSION = 3; /**< You don't have permissions to delete the upload */
```

Comment:
Please update the messages in `src/delagent/ui/DeleteResponse.php` accordingly if changing the codes.